xray/commands/audit/scan.go

```diff
@@ -233,21 +232,15 @@ func (scanCmd *ScanCommand) performScanTasks(fileConsumer parallel.Runner, index
 	vulnerabilities := []services.Vulnerability{}
 	licenses := []services.License{}
 	flatResults := []services.ScanResponse{}
-	tempDirPath, err := fileutils.CreateTempDir()
-	if err != nil {
-		return false, err
-	}
+	var err error
```

Comment:
Let's get this err definition as c,oser as possible to the code lines which use it.